.github/workflows/continous-integration-docker.patch-always.yml

```
@@ -24,3 +24,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - run: 'echo "No build required"'
+
+  lightwalletd-transactions-test:
+    name: lightwalletd tip send
```

Patch job names must match the exact name of the original job, including the calling and called workflow's job names.

This is the name that GitHub checks in the existing branch protection rule:
```suggestion
    name: lightwalletd tip send / Run lwd-send-transactions test
```